webpack.common.js

```diff
@@ -50,5 +51,11 @@ module.exports = {
       filename: '[name].css',
       chunkFilename: '[id].css',
     }),
+    new CopyPlugin({
+      patterns: [
+        { from: "src/assets/audio", to: "assets/audio" },
+        { from: "src/assets/audio", to: "" }
```

Comment:
I think we don't need that.when we have require(something) in scripts then webpack pack music in the dist directory